pkgs/development/python-modules/urllib3/default.nix

```diff
@@ -16,6 +17,8 @@
 , pytestCheckHook
 , tornado
 , trustme
+, enableBrotli ? true
+, enableSecure ? !(stdenv.isAarch64 && stdenv.isDarwin)
```

Comment:
```suggestion
``` 
This is not allowed in pythonPackages and will cause hard to debug errors.

Reply:
I removed the conditionals but happy to add them back defaulting them to `false` if we want to make them opt-in as suggested [here](https://github.com/NixOS/nixpkgs/pull/179159#discussion_r906826976)